state.py

```diff
@@ -42,19 +43,26 @@ def do_automation(self, device, capture):
         device.tap_point(self.points['retry'])
 
 
+#(x1, y1, x2, y2) 10, 140, 40, 780
 states = {
     PlayAttackState(rects=[(10, 780, 140, 40), ], points={'damage': (300, 900), 'attack_speed': (600, 900)}),
     PlayDefenseState(rects=[(10, 780, 140, 40), ]),
     PlayUtilityState(rects=[(10, 780, 140, 40), ]),
     GameOverState(rects=[(180, 1000, 100, 30), ], points={'retry': (230, 1020)})
 }
```

Comment:
I added more points to each state, and set up a 'strategy'. My version always switches to health and buys Health upgrades only. 
I think this is called the 'vampire strategy' on the wiki and discord. 
I guess a small GUI could be created with tkinter to allow easier interaction with the python scripts. 
The GUI could offer multiple 'strategies', i.e. the 'Turtle' strategy for complete beginners.